restaurant/src/assets/home.css

```diff
@@ -0,0 +1,45 @@
+.home-page {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  background-image: url("bg1.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  min-height: 100%;
+}
+.Homeheader {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 10px;
+  font-weight: 700;
+  font-size: 4.5rem;
+  color: rgba(46, 125, 50, 0.5);
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```

Comment:
You do not need to specify font family for each item, it cascades down from the top item.